docs/pages/es/architecture.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: Arquitectura de Meshery 
+permalink: es/architecture
+---
+
+## Arquitectura
+
```

Comment:
Will you incorporate this new [Languages section](https://meshery.layer5.io/docs/architecture#languages), too?